src/exporthtml.ts

```diff
@@ -36,6 +36,7 @@ function generateTranscript<T extends ReturnTypes>(messages: discord.Message[],
 
     document.getElementById('guildname')!.textContent = channel.guild.name;
     document.getElementById('ticketname')!.textContent = channel.name;
+    document.getElementById('tickettopic')!.textContent = `This is the start of the #${channel.name} channel. ${channel.topic  || ''}`;
```

Comment:
I would pass channel.topic into formatContent (if it works) since you can use formatting in topics.
If you do that, you will need to set innerHTML instead of .textContent

Reply:
On it.

Reply:
Should be resolved.